python/lsst/sims/catUtils/supernovae/snObject.py

```diff
@@ -420,6 +419,49 @@ def mwEBVfromMaps(self):
             equatorialCoordinates=self.skycoord)[0]
         return
 
+
+    def redshift(self, z, cosmo):
+        """
+        Redshift the instance holding the intrinsic brightness of the object
+        fixed. By intrinsic brightness here, we mean the BessellB band asbolute
+        magnitude in rest frame. This requires knowing the cosmology
+
+
+        Parameters
+        ----------
+        z : float, mandatory
+            redshift at which the object must be placed.
+        cosmo : instance of `astropy.cosmology` objects, mandatory
+            specifies the cosmology.
+        Returns
+        -------
+        None, but it changes the instance
+
+        """
+        import types
+
+        # Check that the input redshift is a scalar
+        try:
+            assert isinstance(z, types.FloatType) or \
```

Comment:
That would definitely be a better one. I have used this before but was having a hard time finding the exact syntax to do it.